octomap:
    # 0: color tree, 1: max tree, 2: bayes tree
    tree_type: 2
    max_pointcloud_topic: "/semantic_cloud_max"
    bayes_pointcloud_topic: "/semantic_cloud_bayes"
    scout_status: "/scout_status"
    world_frame_id: "/camera_init"
    resolution: 0.10
    max_range: 15.0
    raycast_range: 2.0
    clamping_thres_min: 0.12
    clamping_thres_max: 0.97
    occupancy_thres: 0.5
    prob_hit: 0.7
    prob_miss: 0.4
    save_path : "~/map.bt"
    outrem_radius: 1.0
    outrem_neighbors: 10
